pylintrc

```diff
@@ -0,0 +1,43 @@
+[MASTER]
+# Use multiple processes to speed up Pylint.  It saves only couple of 6 seconds,
+# but the only thing it cannot fetch is duplicate-code, which was mostly
```

Comment:
"cannot fetch"?
I think I understand what you're trying to express here, but the wording does not make much sense to me.

Reply:
It's supposed to be "cannot catch", not idea how that one got there.